base/README.md

```diff
@@ -92,21 +93,21 @@ If you want to access an existing `KinAccount` with options to send `KinPayment`
 ```kotlin
 .useExistingAccount(KinAccount.Id("GATG_example_and_fake_key"))
 ```
-*Note: this variant requires that the `KinEnvironment` knows about this `KinAccount`s `Key.PrivateKey` which can be imported the first time by:*
+*Note: this variant requires that the `KinEnvironment` knows about this `KinAccount`'s `Key.PrivateKey` which can be imported the first time by:*
 ```kotlin
 environment.importPrivateKey(Key.PrivateKey("SDR2_example_and_fake_key"))
     .then { success: Boolean ->
         // Completed
     }
 ```
 ## Read Only Account Data
-For `KinAccount`s that you do **not** have a `Key.PrivateKey` for and only desired read-only access to the data associated with that account use:
+For `KinAccount`s that you do **not** have a `Key.PrivateKey` for and only desired read-only access to the data associated with that account:
 ```kotlin
 .useExistingAccountReadOnly(KinAccount.Id("GATG_example_and_fake_key"))
 ```
 
 ## Sending Payments
-Sending `KinPayment`s are easy. Just add the amount and the destination `KinAccount.Id`.
+Sending `KinPayment`s is easy. Just add the amount and the destination `KinAccount.Id`.
```

Comment:
as KinPayments is plural, I believe "are" is the correct one. However, thanks for the other corrections!

Reply:
This is a weird one, but "is" is the correct verb here. The subject of the sentence is actually "Sending" with "KinPayments" being a modifier on the subject. Because the thing that "is easy" is the act of sending, it should be the singular form "is."